rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -60,6 +60,19 @@ def tearDownClass(cls):
     def test_ppo_compilation_and_schedule_mixins(self):
         """Test whether PPO can be built with all frameworks."""
 
+        model_config = {
+            "torch": dict(
```

Comment:
I don't support all of the model config kwargs that the PPO torch rl module does since I don't have support for lstms.

I also don't have defaults for things like vf_share_layers, activations, and number of hiddens. I want these to be configured by the user explicitly.

Reply:
I mean for torch only. They are supported, at least on the RLModule side of things. lstm is false in the test, but all we had to do once fully supported was to add lstm=True to the test. Now the config does not match what we originally had on master.